src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/CustomCodeActions.cs

```diff
@@ -15,7 +15,7 @@ internal abstract class SimpleCodeAction : CodeAction
             public SimpleCodeAction(string title, string equivalenceKey)
             {
                 Title = title;
-                EquivalenceKey = equivalenceKey;
+                EquivalenceKey = equivalenceKey ?? title;
```

Comment:
❗ This will silently produce a localized equivalence key, which may or may not work as expected.

Reply:
Yes, it is also leading to bunch of test failures. I am reverting this change.